sections/sections-app/src/java/org/sakaiproject/tool/section/jsf/backingbean/EditStudentSectionsBean.java

```diff
@@ -138,10 +143,11 @@ public void init() {
 					
 			boolean member = isEnrolledInSection(enrolled, section);
 			boolean memberOtherSection = isEnrolledInOtherSection(enrolled, section);
-			
-			StudentSectionDecorator decoratedSection = new StudentSectionDecorator(
-					section, catName, taNames, totalEnrollments, member, memberOtherSection, showNegativeSpots);
-			
+
+			StudentSectionDecorator decoratedSection = new StudentSectionDecorator(section, catName, taNames,
+					totalEnrollments, member, memberOtherSection, showNegativeSpots,
+					(taUids.contains(getUserUid()) || (intructorUids.contains(getUserUid()))));
```

Comment:
What if we are the superuser (admin)? 

CC: @ern

Reply:
You are right. If you are superadmin now you can't manage groups. I have created a SAK to fix this

https://sakaiproject.atlassian.net/browse/SAK-47854

Reply:
Meanwhile you can use the "Become user" tool to login as an Instructor